translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -5237,7 +5237,7 @@ template<> Value* SPIRVToLLVM::transValueWithOpcode<OpVariable>(
         }
     }
 
-    const uint32_t addrSpace = pPtrType->getPointerAddressSpace();
+    uint32_t addrSpace = pPtrType->getPointerAddressSpace();
```

Comment:
The pointer type returned by `transType` should already have the correct address space. Otherwise, I would worry that there could be inconsistencies in the applied LLVM types later on, e.g. when passing the OpVariable's value around. So if an address space fixup is required, it should better happen in `transType`.